waspstation/code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -0,0 +1,19 @@
+/datum/reagent/calcium
+	name = "Calcium"
+	description = "A dull gray metal important to bones."
+	reagent_state = SOLID
+	color = "#68675c"
+	metabolization_rate = REAGENTS_METABOLISM
+
+/datum/reagent/ash_fibers
+	name = "Ashen Fibers"
+	description = "Ground plant fibers from a cave fern. Useful for medicines"
+	reagent_state = SOLID
+	color = "#5a4f42"
+	taste_mult = 0
+
+/datum/reagent/titanium
+	name = "Titanium"
+	description = "A light, reflective grey metal used in ship construction"
```

Comment:
missing period

Reply:
extra note: could you add a titanium solidification reaction like the other solidification reactions?

Reply:
ae